project/Dependencies.scala

```diff
@@ -34,6 +34,7 @@ object Dependencies {
     "software.amazon.awssdk" % "lambda" % Versions.aws,
     "software.amazon.awssdk" % "cloudformation" % Versions.aws,
     "software.amazon.awssdk" % "sts" % Versions.aws,
+    "software.amazon.awssdk" % "ssm" % Versions.aws,
```

Comment:
minor point in terms of this PR, but given there are only one file that directly depends on this lib, and there are so many other deps that are needed be other things, have you considered splitting magentaLib into separate sbt projects each with more fine grained dependencies (but all still building into one artifact)